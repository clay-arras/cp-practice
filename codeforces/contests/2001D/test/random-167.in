1
17
23 6 15 21 6 24 19 23 18 2 5 24 1 10 18 21 8
