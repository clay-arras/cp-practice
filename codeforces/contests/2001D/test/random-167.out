12
23 6 15 19 18 2 5 24 1 10 21 8
